src/js/utils/events.js

```diff
@@ -1,11 +1,10 @@
 /**
- * @file events.js
- * @module events
- *
- * Event System (John Resig - Secrets of a JS Ninja http://jsninja.com/)
+ * @file events.js. An Event System (John Resig - Secrets of a JS Ninja http://jsninja.com/)
```

Comment:
Ahhhh, I was wondering what we could do about this. I had a less appealing fix. 👍